src/app/features/mushaf/pages/sura-view/sura-view.page.less

```
@@ -0,0 +1,81 @@
+  &__toolbar {
+    display: flex;
+    justify-content: center;
+    margin-top: 1rem;
+  }
+
+  &__text {
+    padding: 1.5rem;
+    border: 1px solid var(--color-border, #e5e7eb);
+    border-radius: var(--radius-lg, 14px);
+    background: var(--color-surface, #fff);
+    font-family: 'Amiri', 'Scheherazade New', 'Traditional Arabic', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Amiri" (font-family-name-quotes)

(font-family-name-quotes)